src/lib/sw_engine/tvgSwRaster.cpp

```diff
@@ -641,7 +641,8 @@ static bool _rasterUpScaledRleRGBAImage(SwSurface* surface, const SwImage* image
 static bool _transformedRleRGBAImage(SwSurface* surface, const SwImage* image, const Matrix* transform, uint32_t opacity)
 {
     Matrix itransform;
-    if (transform && !mathInverse(transform, &itransform)) return false;
```

Comment:
@p transform can be nullptr.... need to check it as it was before.

Also, we can ignore the initialization of the matrix here.

Otherwise...
if (transform) mathInverse()
else mathIdentity()

Reply:
I'm sorry I misread transform and itransform.